docs/SUMMARY.md

```diff
@@ -108,4 +108,5 @@
 * [Development guide](project/development-guide.md)
 * [Versioning policy](project/versioning-policy.md)
 * [Release process](project/release-process.md)
+* [Maintainer Docs](project/maintainers.md)
```

Comment:
might be better to instead have this as a sub-section of the development guide

Reply:
fixed